app/(pages)/_components/products/_components/ProductCard/ProductCard.module.scss

```diff
@@ -0,0 +1,85 @@
+.productCard {
+    width: 350px;
```

Comment:
Try not to hardcode the card size! This should probably be width: 100%. It should infer its width from the parent grid of cards.